composer.json

```diff
@@ -1,33 +1,36 @@
 {
-    "name": "arquivei/laravel-prometheus-exporter",
-    "description": "A Prometheus exporter for Laravel and Lumen",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Arquivei"
-        }
-    ],
-    "require": {
-        "php": "^7.2.5 || ^8.0",
-        "illuminate/support": "^6.0 || ^7.0 || ^8.0",
-        "illuminate/routing": "^6.0 || ^7.0 || ^8.0",
-        "promphp/prometheus_client_php": "^2.2.1",
-        "guzzlehttp/guzzle": "^6.3 || ^7.0"
-    },
-    "autoload": {
-        "psr-4": {
-            "Arquivei\\LaravelPrometheusExporter\\": "src/",
-            "Arquivei\\LaravelPrometheusExporter\\Tests\\": "tests/"
-        }
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "1.0-dev"
-        }
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^8.0 || ^9.0",
-        "mockery/mockery": "^1.3.3",
-        "orchestra/testbench": "^4.0 || ^5.0 || ^6.0"
+  "name": "arquivei/laravel-prometheus-exporter",
+  "description": "A Prometheus exporter for Laravel and Lumen",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Arquivei"
     }
+  ],
+  "require": {
+    "php": "^8.0 || ^8.1",
```

Comment:
As you've removed all previous version until 8.0, I believe we can leave the constraint ~8.0 here, can't we?

Reply:
This might make sense, but writing it this way says that version 8.2 will be accepted (when it's released) even if we don't have the necessary tests yet.

Reply:
Indeed